### ISSUE TITLE
Should we specify a limiting max heap size on test jobs to control things like max dump sizes ?

### ISSUE THREAD

**Issue by andrew-m-leonard** [AdoptOpenJDK/openjdk-tests#2195]
Issues have been seen with openj9 AIX testcases that produce dumps that are 30+Gb, because the default heapsize is 25% available RAM, and on some AIX boxes there can be a huge amount of available RAM. But we may only have a 30Gb disk partition.

Should we specify a -Xmx, at least for openj9?

**Comment by llxia** [comment]
IMHO, each test should be responsible for itself. Having the test pipeline blindly overrides the value may cause various problems.

**Comment by smlambert** [comment]
Yes, I think we saw that when trying to limit -Xmx during the release... that some tests need different values and we'll see different failures if blanket change is applied.  

I think we can better handle the situation this issue is intended to report by a combination of:
- tuning the particular functional tests in question (it will be useful to list those as part of the description of this issue)
- the MachineInfo work that is now underway in TKG [Testing environment checks issue](https://github.com/AdoptOpenJDK/TKG/issues/45).

**Comment by andrew-m-leonard** [comment]
Here's a list of the functional tests changewd: https://github.com/eclipse/openj9/pull/11340
